packages/truffle-contract/lib/execute.js

```diff
@@ -7,6 +7,16 @@ var Reason = require("./reason");
 var handlers = require("./handlers");
 var override = require("./override");
 var reformat = require("./reformat");
+var BN = require("bn.js");
+
+function createBN(n) {
```

Comment:
I think `createBN` should be moved into `truffle-contract/lib/utils.js`
(would also remove the redundancy of declaring the func twice)

Reply:
Ah good catch. I wasn't even thinking that they were in the same package :sweat_smile: